subsys/bluetooth/host/Kconfig

```diff
@@ -155,10 +155,22 @@ config BT_RX_STACK_SIZE
 
 config BT_RX_PRIO
 	# Hidden option for Co-Operative Rx thread priority
+	# HCI drivers that select BT_RECV_IS_RX_THREAD should call bt_recv from
+	# this priority.
+	# Otherwise the host will process events in this priority.
 	int
 	depends on BT_HCI_HOST || BT_RECV_IS_RX_THREAD
 	default 8
 
+config BT_RX_PRIO_HIGH
```

Comment:
I believe the important detail here is to emphasize that this thread runs in a higher priority than BT_RX_PRIO. Using "HIGH" makes that more clear. What about BT_DRIVER_RX_PRIO_HIGH?

Reply:
`BT_DRIVER_RX_HIGH_PRIO` general convention in the zephyr tree to use suffix of `_PRIO` for priority value

Reply:
This will be a Kconfig style exception compared to others like BT_HCI_TX_PRIO, BT_TX_PRIO, NET_INIT_PRIO, NET_CONFIG_INIT_PRIO etc... all of which end with _PRIO for priority value. `BT_DRIVER_RX_PRIO` is more in sync with explanation in the added comments.

Reply:
Sorry, I thought I had force-pushed, but apparently not.
It is updated, to what you suggested.